gwswlib/scripts.py

```diff
@@ -46,6 +56,21 @@ def run_import_export(
     else:
         raise OptionException("no available export type %r is selected" % export_type)
 
+    if log_relpath is not None:
+        logger.info("Log file with warnings and errors is created %r", log_relpath)
+
+    logger.info("Exchange of GWSW-hydx finished")
+
+    return "method is finished"
+
+
+def write_logging_to_file(log_relpath):
```

Comment:
De logging setup zit al in de `def main()`? Daar wordt ook naar `--verbose` gekeken qua log level.
Daar wordt het met `.basicConfig()` gedaan, je kan dat veranderen naar wat je hier doet.

Wat vaak handig is: het output loglevel voor de file lager zetten dan voor de commandline output. (file op DEBUG, console op INFO, of INFO/WARN).

In ieder geval: doe het niet op twee plekken anders :-)